src/app/shared/stache/directives/stick-to-top.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { Directive, ElementRef, HostListener, Renderer } from '@angular/core';
+
+import { SkyAppWindowRef } from '@blackbaud/skyux-builder/runtime';
```

Comment:
We'll have to remember to update this once @Blackbaud-SteveBrush 's PR gets merged.